test/plugins/builds.test.js

```diff
@@ -1619,12 +1623,16 @@ describe('build plugin test', () => {
 
                     // for chainPR settings
                     pipelineMock.chainPR = true;
-                    eventMock.pr = { ref: 'pull/15/merge' };
+                    eventMock.pr = { ref: 'pull/15/merge', prSource: 'branch', prInfo: {} };
```

Comment:
Is `prInfo` blank in this case? I think that `prInfo` has some values in the PR events.

Reply:
Thank you review.
I forgot to fix prInfo.
so, I add a value inside prInfo.